ocs_ci/utility/reporting.py

```diff
@@ -73,6 +73,10 @@ def get_rp_launch_attributes():
         rp_attrs["fips"] = True
     if config.ENV_DATA.get("encryption_at_rest"):
         rp_attrs["encryption_at_rest"] = True
+    if config.RUN["skipped_on_ceph_health_ratio"] > 0:
+        rp_attrs["ceph_health_skips"] = True
+    if config.RUN["skipped_on_ceph_health_ratio"] > 0.25:
```

Comment:
Now we have this value on two places, I think that this value should be configurable from config. In default config we can have this set to 0.25 or any other number we will agree on and even add some CLI parameter which will increase this threshold. I saw the concerns in the email. This might even allow us to change default to 0% threshold, and by CLI or config change it to some other value if needed. WDYT?

Reply:
I agree with Petr, that it will be useful to make the threshold configurable and also tend to propose to set it for 0% by default.

Reply:
added `config.RUN["skipped_on_ceph_health_threshold"]` which is set to 0.25 (25%) by default.
In addition to that, I decided that for `acceptance` suite, the value of `config.RUN["skipped_on_ceph_health_threshold"]` would be set to 0 so any skip on Ceph health during an `acceptance` suite execution would cause `test_report_skip_triggering_test` to fail